apps/chat/src/components/Chat/ChatMessage/ChatMessageContent.tsx

```diff
@@ -409,7 +431,19 @@ export const ChatMessageContent = ({
                     selectedDialLinks.length > 0) && (
                     <div className="mb-2.5 grid max-h-[100px] grid-cols-1 gap-1 overflow-auto sm:grid-cols-2 md:grid-cols-3">
                       <ChatInputAttachments
-                        files={newEditableAttachments}
+                        files={newEditableAttachments.filter(
+                          (f) =>
+                            f.contentType !== FOLDER_ATTACHMENT_CONTENT_TYPE,
+                        )}
+                        folders={
+                          canAttachFolders
+                            ? (newEditableAttachments.filter(
+                                (f) =>
+                                  f.contentType ===
+                                  FOLDER_ATTACHMENT_CONTENT_TYPE,
+                              ) as unknown as FileFolderInterface[])
+                            : undefined
+                        }
```

Comment:
Maybe for the readability it is better to extract all this lines into the variable?
What do you think?